bridge/contracts/Bridge/Bridge.sol

```diff
@@ -158,17 +158,17 @@ contract Bridge is Initializable, IBridge, IERC777Recipient, UpgradablePausable,
 
 	function knownToken(uint256 chainId, address originalToken) public view returns(bool) {
 		// specification for retrocompatibility
-		if (isChain(chainId)) {
-			bool knowToken = deprecatedKnownTokens[originalToken];
-			if (knowToken) {
-				return knowToken;
+		if (isCurrentChainId(chainId)) {
+			bool knownTokenValue = deprecatedKnownTokens[originalToken];
+			if (knownTokenValue) {
+				return knownTokenValue;
 			}
 		}
 		return knownTokenByChain[chainId][originalToken];
 	}
 
-	function setKnownTokenByChain(uint256 chainId, address originalToken, bool knowToken) public {
-		knownTokenByChain[chainId][originalToken] = knowToken;
+	function setKnownTokenByChain(uint256 chainId, address originalToken, bool knownTokenValue) public {
```

Comment:
this might also need to have the `onlyOwner` modifier, right?

Reply:
(just a question)

Reply:
Actually, I think this one has to be public because it is called by the `tokensReceived` function